types/jquery/index.d.ts

```diff
@@ -25,14 +25,14 @@
 // TypeScript Version: 2.3
 
 declare module 'jquery' {
-    function factory(window: Window): JQueryStatic;
+    function factory(window: Window): JQueryStatic & JQuery;
```

Comment:
The factory function definitely does not return an _instance_ of `JQuery`.

```ts
import jsdomify from 'jsdomify'
jsdomify.create()

import jqueryFactory = require('jquery')
jqueryFactory(window)(window).data('foo') // returns undefined
jqueryFactory(window).data('foo') // would validate with this change, but throws
```

The case where jquery is loaded on a real DOM is already correctly handled by the `factoryOrJQuery` type.